### PR TITLE
Made PGD Cookbook compatible with PGD Dev Branch

### PR DESCRIPTION
**Pull request by ElijahCaine**
Refs #9

- Replaced django_settings.py.erb with modified version of current prod
settings.py.
    > All 'secrets' from old django settings file are translated over
    to new settings file.
- Changed attributes/default.rb repo and revision to be the pgd repository
and development branch.
- Modified reipes/default.rb config_file path from /pgd/settings.py to
/settings.py

### CODE REVIEW

File: templates/default/django_settings.py.erb [hunk 1]
```diff
@@ -41,142 +43,82 @@ LANGUAGE_CODE = 'en-us'
 
 SITE_ID = 1
 
+#Google analytics ID. Enter the full id, as in: UA-xxxxxx-x 
+GOOGLE_ID = 'UA-9880071-1'
```
Comment (jordane) [inline templates/default/django_settings.py.erb]: You set this to `None` above.

Comment (mathuin) [inline templates/default/django_settings.py.erb]: It's that way in production, too, actually, which was the source for this document.

The correct default value is None, of course.  What other values should we be setting in the template?

Comment (jordane) [inline templates/default/django_settings.py.erb]: Anything you change from the default.

Comment (ElijahCaine) [inline templates/default/django_settings.py.erb]: @mathuin What value should this be set too? I can add it to the data-bag if we want to do that now.

Comment (mathuin) [inline templates/default/django_settings.py.erb]: If you can put the full ID in the databag (the 'UA-...' string) that would be great!

**Comment by ramereth** [comment]
This needs to be rebased on master as its not mergable right now.

**Comment by mathuin** [comment]
The point of this pull request is for this branch of the cookbook to be compatible with the develop branch of PGD.  It's okay that this pull request is not mergable at the present time -- it is intended to be merged when PGD version 1.1 is released, which is when the PGD develop branch will be merged back into the PGD master branch.

**Comment by ramereth** [comment]
We need to account for changes that happen for master no matter what happens on the 1.1 side of things IMO. But I'll let @jordane chime in if he feels differently.

**Comment by ramereth** [comment]
A few things I notice off the bat:

1. Why are we starting apache in default.rb? The apache cookbook should be doing that...
2. IMO we should be creating a pgd system user and set a default location to install the app no matter if we're on vagrant/openstack/whatever. Something like /opt/pgd might work or something in /var/lib/pgd.
3. Its good to add ``.rubocop.yml`` and run it to keep your code up to style standards. Here's an example file you can use:

``` yml
AllCops:
  Include:
    - '**/Berksfile'
    - '**/Cheffile'
  Exclude:
    - 'metadata.rb'
Lint/AmbiguousRegexpLiteral:
  Exclude:
    - 'test/integration/**/*.rb'
```
4. You might run ``berks update`` and see what happens. It will likely break because the newer mysql cookbook has different dependencies. I already mentioned this in #14 but we may want to put the fix here instead. For now we should match the version we have in production since that's what we're using but we'll eventually want to use the newer one. Right now the test kitchen run breaks on an updated berk install.
5. Running ``foodcritic .`` is also a good thing to do to catch chef specific linting problems.

Looks mostly good though! Let's try and get #14 merged ASAP and then rebase this on that

**Comment by jordane** [comment]
@ramereth I don't think we want to exclude `test/integration/**/*.rb` from rubocop..

**Comment by ramereth** [comment]
@jordane look at it more closely. That's ONLY excluding the regex tests that we want to keep anyways because of server spec's way of doing things.

**Comment by jordane** [comment]
@ramereth Ah, derp. I can't read.

**Comment by ElijahCaine** [comment]
@ramereth I have created issues for the first two points you made above and have created an issue for including a .rubocop.yml. The rest of your comments I will complete and push asap. The issues should be resolved before this pull request is completed.

**Pull request merged**